ex2-web-scrapping/browser.py

```diff
@@ -0,0 +1,63 @@
+import os
```

Comment:
General Comments:
- use prospector and pydocstyle to check your code
- Write comments with google style on every function